rest/access_test.go

```diff
@@ -603,19 +631,19 @@ func TestAllDocsAccessControl(t *testing.T) {
 	assert.NoError(t, a.Save(alice))
 
 	// Get a single doc the user has access to:
-	request, _ := http.NewRequest("GET", "/db/doc3", nil)
+	request, _ := http.NewRequest("GET", "/db."+s+"."+c+"/doc3", nil)
 	request.SetBasicAuth("alice", "letmein")
 	response := rt.Send(request)
 	RequireStatus(t, response, 200)
```

Comment:
you can change this to:

```
response := rt.SendUserRequest(http.MethodGet, "/{{.keyspace}}/doc3", nil, "alice")
rest.RequireStatus(response, http.StatusOK)
```

This was presumably written before we had a rest helper function